### PR TITLE
[FEATURE] isOpen() function for formatted file.

### PR DESCRIPTION
**Pull request by smehringer**
suggestion to #1834

@rrahn what do you think?

### CODE REVIEW

File: include/seqan/stream/formatted_file.h [hunk 1]
```diff
@@ -716,6 +716,15 @@ inline bool open(FormattedFile<TFileFormat, TDirection, TSpec> & file,
 }
 
 // ----------------------------------------------------------------------------
+// Function isOpen()
+// ----------------------------------------------------------------------------
+template <typename TFileFormat, typename TDirection, typename TSpec>
+inline bool isOpen(FormattedFile<TFileFormat, TDirection, TSpec> & file)
```
Comment (rrahn) [inline include/seqan/stream/formatted_file.h]: Should be ```const &```

Comment (rrahn) [inline include/seqan/stream/formatted_file.h]: And add documentation, in case we have the formatted file documented.

Comment (rrahn) [inline include/seqan/stream/formatted_file.h]: 👍 😄

**Comment by h-2** [comment]
+1

PS: The last commit should have had `[TEST] ...` label, only the first one was `[FEATURE] ...` :nail_care:

**Pull request merged**